src/components/Ranking.tsx

```
@@ -1,4 +1,5 @@
 import styled from "@emotion/styled";
+import { AuthorTag } from "./Common/Tag/AuthorTag";
```

ESLint

[error] 'AuthorTag' is defined but never used.

(@typescript-eslint/no-unused-vars)